legacy/src/capi-multiple-hosted/web/index.scss

```diff
@@ -28,3 +28,7 @@
     fill: #ffffff;
   }
 }
+
+body {
```

Comment:
Entirely your call but is there scope to create a shared `_merchandising` CSS file from which any shared styling can be imported?

Reply:
Yep that's probably a good shout! I'll update accordingly